apps/docs/docs/user/intro/intro.md

```diff
@@ -23,6 +27,10 @@ npm install -g @jvalue/jayvee-interpreter@0.0.17
 
 Updating the interpreter is done by reinstalling it using `npm`. Make sure to also update the [VSCode plugin](#vscode-plugin) to match the installed interpreter if you use it.
 
+IMPORTANT: If a new Jayvee update is available, you need to update your VSCode extension for Jayvee as well.
+
+More details on updating can be found [here](./update.md).
+
 ```bash
 npm install -g @jvalue/jayvee-interpreter
 ```
```

Comment:
The point of this was to move all the update docs to one place, not keep it around in two places. So here, please just link to the update page and remove the rest. E.g.

```suggestion
Details about how to update Jayvee and the VSCode extension can be found [here](./update.md).
```